src/procedure/nodes/importCoordinates.cpp

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#include "procedure/nodes/importCoordinates.h"
+#include "classes/configuration.h"
+#include "keywords/fileAndFormat.h"
+
+ImportCoordinatesProcedureNode::ImportCoordinatesProcedureNode()
+    : ProcedureNode(ProcedureNode::NodeType::ImportCoordinates, {ProcedureNode::GenerationContext})
+{
+    keywords_.setOrganisation("Options", "Source Data");
+    keywords_.add<FileAndFormatKeyword>("File", "File / format for initial coordinate to read in", fileSource_, "EndFile");
+}
+
+/*
+ * Identity
+ */
+
+// Return whether a name for the node must be provided
+bool ImportCoordinatesProcedureNode::mustBeNamed() const { return false; }
+
+/*
+ * Execute
+ */
+
+// Prepare any necessary data, ready for execution
+bool ImportCoordinatesProcedureNode::prepare(const ProcedureContext &procedureContext)
+{
+    if (!fileSource_.hasFilename())
+        return Messenger::error("A suitable coordinate file and format must be supplied.\n");
+    if (!fileSource_.fileExists())
+        return Messenger::error("Specified coordinate file to import '{}' doesn't exist\n", fileSource_.filename());
+    return true;
+}
+
+// Execute node
+bool ImportCoordinatesProcedureNode::execute(const ProcedureContext &procedureContext)
+{
+    auto *cfg = procedureContext.configuration();
+    return fileSource_.importData(cfg, &procedureContext.processPool());
```

Comment:
Don't really need the temporary
```suggestion
    return fileSource_.importData(procedureContext.configuration(), &procedureContext.processPool());
```